cmd/juju/controller/destroy.go

```diff
@@ -54,6 +54,7 @@ type destroyCommand struct {
 	destroyModels  bool
 	destroyStorage bool
 	releaseStorage bool
+	credentialName string
```

Comment:
I am not too sure that this is a good idea... We have measures to ensure that your user has permissions to destroy a model. I am not sure that we want to a different cloud credential than the one that controller is using... Especially since I am not convinced that we have a check that this new credential will have access to all controller "assets"...